crypto/rsa/rsa.c

```diff
@@ -121,23 +121,17 @@ int rsa_new_end(RSA *rsa, BN_CTX *ctx) {
    * BN_FLG_CONSTTIME flag. */
```

Comment:
I'm a little confused what you want me to do for this. Which variables are you referring to here?